README.md

```diff
@@ -37,6 +37,16 @@ See the [Installation Guide](https://docs.securedrop.org/en/stable/#installtoc).
 
 See our [contribution page](CONTRIBUTING.md)
 
+## Developer Quickstart
+
+Ensure you have Docker installed and:
+
+```
+make dev
+```
+
+This will start the source interface on `127.0.0.1:8080` and the journalist interface on `127.0.0.1:8081`.
+
```

Comment:
Maybe another line mentioning that the credentials to use those interfaces will be printed in the terminal?